mhctools/parsing.py

```diff
@@ -477,3 +477,53 @@ def parse_netmhciipan_stdout(
         ic50_index=8,
         rank_index=9,
         score_index=7)
+
+def parse_netmhciipan4_stdout(
+        stdout,
+        prediction_method_name="netmhciipan",
+        sequence_key_mapping=None,
+        mode="elution_score"):
+    """
+    # Threshold for Strong binding peptides (%Rank) 2%
+    # Threshold for Weak binding peptides (%Rank)   10%
+
+    # Allele: DRB1_0101
+    --------------------------------------------------------------------------------------------------------------------------------------------
+     Pos           MHC              Peptide   Of        Core  Core_Rel        Identity      Score_EL %Rank_EL Exp_Bind      Score_BA  Affinity(nM) %Rank_BA  BindLevel
+    --------------------------------------------------------------------------------------------------------------------------------------------
+       1     DRB1_0101      PAPAPSWPLSSSVPS    4   PSWPLSSSV     0.327            test      0.000857    79.79       NA      0.327674       1442.91    54.35       
+       2     DRB1_0101      APAPSWPLSSSVPSQ    3   PSWPLSSSV     0.333            test      0.001268    71.87       NA      0.346949       1171.30    50.15       
+       3     DRB1_0101      PAPSWPLSSSVPSQK    4   WPLSSSVPS     0.713            test      0.002836    54.45       NA      0.412004        579.40    36.66       
+       4     DRB1_0101      APSWPLSSSVPSQKT    3   WPLSSSVPS     0.773            test      0.003677    49.14       NA      0.448939        388.53    29.75       
+       5     DRB1_0101      PSWPLSSSVPSQKTY    2   WPLSSSVPS     0.407            test      0.001602    66.79       NA      0.470979        306.09    25.98       
+       6     DRB1_0101      SWPLSSSVPSQKTYQ    3   LSSSVPSQK     0.633            test      0.001671    65.82       NA      0.476222        289.21    25.07       
+       7     DRB1_0101      WPLSSSVPSQKTYQG    3   SSSVPSQKT     0.553            test      0.001697    65.45       NA      0.447217        395.83    30.05    
+
+
+    """
+    check_stdout_error(stdout, "NetMHCIIpan")
+
+    if mode not in ["elution_score", "binding_affinity"]:
+        raise ValueError("Mode is %s but must be one of: elution_score, binding affinity" % mode)
```

Comment:
what about the `all_output` mode?

Reply:
I accidentally left that in elsewhere in this PR - thought this would be a separate mode to output the whole dataframe, but that'll be in a different PR. Removing it